config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -120,6 +120,45 @@ periodics:
     testgrid-alert-email: skclark@redhat.com
     description: "Uses kubetest to run serial node-e2e tests (+Serial, -Flaky|Benchmark|Node*Feature)"
 
+- name: ci-kubernetes-node-arm64-ubuntu-serial
+  interval: 4h
+  cluster: k8s-infra-prow-build
+  annotations:
+    testgrid-dashboards: sig-node-kubelet
+    testgrid-tab-name: kubelet-gce-e2e-arm64-ubuntu-serial
+    description: "Run serial node e2e tests on ARM64 environment on Ubuntu"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
```

Comment:
This will likely need 

```
preset-dind-enabled: "true"
```

because ARM build needs to use dockerized build so we need to enable docker-in-docker.

Reply:
Thanks for the pointer, added.